src/domain/documentRequestFactory.ts

```diff
@@ -0,0 +1,71 @@
+import AWS, { S3 } from 'aws-sdk';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import getCertificate from './getCertificate';
+import getLetter from './getLetter';
+import getPlate from './getPlate';
+
+const {
+  NODE_ENV, BUCKET, BRANCH,
+} = process.env;
+
+export default async (vin: string, testNumber: string, plateSerialNumber: string, systemNumber: string): Promise<APIGatewayProxyResult> => {
+  const s3 = new S3(
+    process.env.IS_OFFLINE && {
+      s3ForcePathStyle: true,
+      // You will need to create your s3local profile (~/.aws/credentials) if you are not using any
+      accessKeyId: 'S3RVER',
+      secretAccessKey: 'S3RVER',
+      endpoint: new AWS.Endpoint('http://localhost:4569'),
+    },
+  );
+
+  const isCertificate = vin && testNumber && !plateSerialNumber && !systemNumber;
+  const isPlate = plateSerialNumber && !vin && !testNumber && !systemNumber;
+  const isLetter = !plateSerialNumber && vin && !testNumber && systemNumber;
+
+  if (isCertificate) {
```

Comment:
Might be better to have a switch instead

Reply:
I generally avoid switches. Any compelling reason to change?